tests/format/flow-repo/predicates-parsing/pass.js

```diff
@@ -17,7 +17,6 @@ var a1 = (x: mixed): %checks => x !== null;
 const insert_a_really_big_predicated_arrow_function_name_here = (x)
   : %checks => x !== null;
 
-declare var x;
```

Comment:
This file supposed to sync with the flow repo, can you copy a new one from there? If it's removed we can move it to `flow` dir.